{"text": "2022-07-14T05:14:22.058370+0300 | ERROR      | An error has been caught in function 'call', process 'MainProcess' (18924), thread 'MainThread' (11496):\nTraceback (most recent call last):\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\bot.py\", line 77, in <module>\n    bot.run(config.get_token)\n    │   │   │      └ <property object at 0x000001957740DAD0>\n    │   │   └ <_config.Config object at 0x0000019575A57A30>\n    │   └ <function Client.run at 0x000001957656D5A0>\n    └ <nextcord.ext.commands.bot.Bot object at 0x000001957726FD90>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 796, in run\n    loop.run_forever()\n    │    └ <function ProactorEventLoop.run_forever at 0x0000019575860D30>\n    └ <ProactorEventLoop running=True closed=False debug=False>\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\windows_events.py\", line 321, in run_forever\n    super().run_forever()\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\base_events.py\", line 600, in run_forever\n    self._run_once()\n    │    └ <function BaseEventLoop._run_once at 0x00000195757AE0E0>\n    └ <ProactorEventLoop running=True closed=False debug=False>\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\base_events.py\", line 1896, in _run_once\n    handle._run()\n    │      └ <function Handle._run at 0x0000019575741900>\n    └ <Handle Task.task_wakeup(<Future finished result=None>)>\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n    │    │            │    │           │    └ <member '_args' of 'Handle' objects>\n    │    │            │    │           └ <Handle Task.task_wakeup(<Future finished result=None>)>\n    │    │            │    └ <member '_callback' of 'Handle' objects>\n    │    │            └ <Handle Task.task_wakeup(<Future finished result=None>)>\n    │    └ <member '_context' of 'Handle' objects>\n    └ <Handle Task.task_wakeup(<Future finished result=None>)>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 445, in _run_event\n    await coro(*args, **kwargs)\n          │     │       └ {}\n          │     └ (<nextcord.interactions.Interaction object at 0x000001957762AA80>,)\n          └ <bound method Client.on_interaction of <nextcord.ext.commands.bot.Bot object at 0x000001957726FD90>>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 1872, in on_interaction\n    await self.process_application_commands(interaction)\n          │    │                            └ <nextcord.interactions.Interaction object at 0x000001957762AA80>\n          │    └ <function Client.process_application_commands at 0x000001957656EE60>\n          └ <nextcord.ext.commands.bot.Bot object at 0x000001957726FD90>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 1892, in process_application_commands\n    await app_cmd.call_from_interaction(interaction)\n          │       │                     └ <nextcord.interactions.Interaction object at 0x000001957762AA80>\n          │       └ <function BaseApplicationCommand.call_from_interaction at 0x00000195764DB2E0>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000195776A72E0>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 2045, in call_from_interaction\n    await self.call(self._state, interaction)  # type: ignore\n          │    │    │    │       └ <nextcord.interactions.Interaction object at 0x000001957762AA80>\n          │    │    │    └ <nextcord.state.ConnectionState object at 0x00000195776A5150>\n          │    │    └ <nextcord.application_command.SlashApplicationCommand object at 0x00000195776A72E0>\n          │    └ <function SlashApplicationCommand.call at 0x00000195764DBBE0>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000195776A72E0>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 2397, in call\n    await self.call_slash(state, interaction, option_data)\n          │    │          │      │            └ [{'value': 'тема', 'type': 3, 'name': 'do'}]\n          │    │          │      └ <nextcord.interactions.Interaction object at 0x000001957762AA80>\n          │    │          └ <nextcord.state.ConnectionState object at 0x00000195776A5150>\n          │    └ <function SlashCommandMixin.call_slash at 0x00000195764DA830>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000195776A72E0>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 1514, in call_slash\n    await self.invoke_callback_with_hooks(state, interaction, **kwargs)\n          │    │                          │      │              └ {'do': 'тема'}\n          │    │                          │      └ <nextcord.interactions.Interaction object at 0x000001957762AA80>\n          │    │                          └ <nextcord.state.ConnectionState object at 0x00000195776A5150>\n          │    └ <function CallbackMixin.invoke_callback_with_hooks at 0x00000195764D9900>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000195776A72E0>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 824, in invoke_callback_with_hooks\n    await self(interaction, *args, **kwargs)\n          │    │             │       └ {'do': 'тема'}\n          │    │             └ ()\n          │    └ <nextcord.interactions.Interaction object at 0x000001957762AA80>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000195776A72E0>\n\n> File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\cogs\\master.py\", line 159, in call\n    return await self.callback(do, inter, guild)\n                 │    │        │   │      └ 'electro'\n                 │    │        │   └ <nextcord.interactions.Interaction object at 0x000001957762AA80>\n                 │    │        └ 'тема'\n                 │    └ <function MasterCommands.callback at 0x00000195776C1A20>\n                 └ <cogs.master.MasterCommands object at 0x00000195776A70A0>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\cogs\\master.py\", line 139, in callback\n    await self.change_topic(inter, guild)  # type: ignore\n          │    │            │      └ 'electro'\n          │    │            └ <nextcord.interactions.Interaction object at 0x000001957762AA80>\n          │    └ <function MasterCommands.change_topic at 0x00000195776C16C0>\n          └ <cogs.master.MasterCommands object at 0x00000195776A70A0>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\cogs\\master.py\", line 86, in change_topic\n    await inter.send(\n          │     └ <function Interaction.send at 0x00000195764876D0>\n          └ <nextcord.interactions.Interaction object at 0x000001957762AA80>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\interactions.py\", line 535, in send\n    return await self.followup.send(\n                 │    └ <nextcord.utils.CachedSlotProperty object at 0x0000019576425030>\n                 └ <nextcord.interactions.Interaction object at 0x000001957762AA80>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\webhook\\async_.py\", line 1523, in send\n    data = await adapter.execute_webhook(\n                 │       └ <function AsyncWebhookAdapter.execute_webhook at 0x000001957647F910>\n                 └ <nextcord.webhook.async_.AsyncWebhookAdapter object at 0x00000195764266E0>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\webhook\\async_.py\", line 206, in request\n    raise HTTPException(response, data)\n          │             │         └ {'code': 50035, 'errors': {'embeds': {'0': {'fields': {'1': {'value': {'_errors': [{'code': 'BASE_TYPE_REQUIRED', 'message': ...\n          │             └ <ClientResponse(https://discord.com/api/v10/webhooks/861626405313708062/aW50ZXJhY3Rpb246OTk2OTYyODE0MDQ2NTE5Mjk2OmxiUFBXVGZUb...\n          └ <class 'nextcord.errors.HTTPException'>\n\nnextcord.errors.HTTPException: 400 Bad Request (error code: 50035): Invalid Form Body\nIn embeds.0.fields.1.value: This field is required\n", "record": {"elapsed": {"repr": "0:00:17.748999", "seconds": 17.748999}, "exception": {"type": "HTTPException", "value": "400 Bad Request (error code: 50035): Invalid Form Body\nIn embeds.0.fields.1.value: This field is required", "traceback": true}, "extra": {}, "file": {"name": "master.py", "path": "C:\\Users\\phili\\PycharmProjects\\GuildsBot\\cogs\\master.py"}, "function": "call", "level": {"icon": "❌", "name": "ERROR", "no": 40}, "line": 159, "message": "An error has been caught in function 'call', process 'MainProcess' (18924), thread 'MainThread' (11496):", "module": "master", "name": "cogs.master", "process": {"id": 18924, "name": "MainProcess"}, "thread": {"id": 11496, "name": "MainThread"}, "time": {"repr": "2022-07-14 05:14:22.058370+03:00", "timestamp": 1657764862.05837}}}
{"text": "2022-07-14T08:04:10.377761+0300 | ERROR      | An error has been caught in function 'get_users', process 'MainProcess' (20440), thread 'MainThread' (20364):\nTraceback (most recent call last):\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\bot.py\", line 77, in <module>\n    bot.run(config.get_token)\n    │   │   │      └ <property object at 0x00000171E9D3DAD0>\n    │   │   └ <_config.Config object at 0x00000171E8387A30>\n    │   └ <function Client.run at 0x00000171E8E9D5A0>\n    └ <nextcord.ext.commands.bot.Bot object at 0x00000171E9B9FD60>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 796, in run\n    loop.run_forever()\n    │    └ <function ProactorEventLoop.run_forever at 0x00000171E8190D30>\n    └ <ProactorEventLoop running=True closed=False debug=False>\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\windows_events.py\", line 321, in run_forever\n    super().run_forever()\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\base_events.py\", line 600, in run_forever\n    self._run_once()\n    │    └ <function BaseEventLoop._run_once at 0x00000171E80DE0E0>\n    └ <ProactorEventLoop running=True closed=False debug=False>\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\base_events.py\", line 1896, in _run_once\n    handle._run()\n    │      └ <function Handle._run at 0x00000171E8071900>\n    └ <Handle <TaskStepMethWrapper object at 0x00000171EDA12AA0>()>\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n    │    │            │    │           │    └ <member '_args' of 'Handle' objects>\n    │    │            │    │           └ <Handle <TaskStepMethWrapper object at 0x00000171EDA12AA0>()>\n    │    │            │    └ <member '_callback' of 'Handle' objects>\n    │    │            └ <Handle <TaskStepMethWrapper object at 0x00000171EDA12AA0>()>\n    │    └ <member '_context' of 'Handle' objects>\n    └ <Handle <TaskStepMethWrapper object at 0x00000171EDA12AA0>()>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 445, in _run_event\n    await coro(*args, **kwargs)\n          │     │       └ {}\n          │     └ (<nextcord.interactions.Interaction object at 0x00000171E9D34EE0>,)\n          └ <bound method Client.on_interaction of <nextcord.ext.commands.bot.Bot object at 0x00000171E9B9FD60>>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 1872, in on_interaction\n    await self.process_application_commands(interaction)\n          │    │                            └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │    └ <function Client.process_application_commands at 0x00000171E8E9EE60>\n          └ <nextcord.ext.commands.bot.Bot object at 0x00000171E9B9FD60>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 1892, in process_application_commands\n    await app_cmd.call_from_interaction(interaction)\n          │       │                     └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │       └ <function BaseApplicationCommand.call_from_interaction at 0x00000171E8E0B2E0>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 2045, in call_from_interaction\n    await self.call(self._state, interaction)  # type: ignore\n          │    │    │    │       └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │    │    │    └ <nextcord.state.ConnectionState object at 0x00000171E9FCA290>\n          │    │    └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n          │    └ <function SlashApplicationCommand.call at 0x00000171E8E0BBE0>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 2397, in call\n    await self.call_slash(state, interaction, option_data)\n          │    │          │      │            └ [{'value': '877604100706025482', 'type': 8, 'name': 'guild_role'}]\n          │    │          │      └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │    │          └ <nextcord.state.ConnectionState object at 0x00000171E9FCA290>\n          │    └ <function SlashCommandMixin.call_slash at 0x00000171E8E0A830>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 1514, in call_slash\n    await self.invoke_callback_with_hooks(state, interaction, **kwargs)\n          │    │                          │      │              └ {'guild_role': <Role id=877604100706025482 name='Cryo'>}\n          │    │                          │      └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │    │                          └ <nextcord.state.ConnectionState object at 0x00000171E9FCA290>\n          │    └ <function CallbackMixin.invoke_callback_with_hooks at 0x00000171E8E09900>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 824, in invoke_callback_with_hooks\n    await self(interaction, *args, **kwargs)\n          │    │             │       └ {'guild_role': <Role id=877604100706025482 name='Cryo'>}\n          │    │             └ ()\n          │    └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n> File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\cogs\\master.py\", line 120, in get_users\n    f\"{self.get_timestamp(minutes=5, discord=True, style='T')}\"\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\slavebot\\utils.py\", line 126, in get_timestamp\n    days: timedelta = timedelta(days=\"\", hours=hours, minutes=minutes, seconds=seconds)\n    │                 │                        │              │                └ 0\n    │                 │                        │              └ 5\n    │                 │                        └ 0\n    │                 └ <class 'datetime.timedelta'>\n    └ ''\n\nTypeError: unsupported type for timedelta days component: str\n", "record": {"elapsed": {"repr": "0:00:07.990002", "seconds": 7.990002}, "exception": {"type": "TypeError", "value": "unsupported type for timedelta days component: str", "traceback": true}, "extra": {}, "file": {"name": "master.py", "path": "C:\\Users\\phili\\PycharmProjects\\GuildsBot\\cogs\\master.py"}, "function": "get_users", "level": {"icon": "❌", "name": "ERROR", "no": 40}, "line": 120, "message": "An error has been caught in function 'get_users', process 'MainProcess' (20440), thread 'MainThread' (20364):", "module": "master", "name": "cogs.master", "process": {"id": 20440, "name": "MainProcess"}, "thread": {"id": 20364, "name": "MainThread"}, "time": {"repr": "2022-07-14 08:04:10.377761+03:00", "timestamp": 1657775050.377761}}}
{"text": "2022-07-14T08:04:10.484756+0300 | ERROR      | An error has been caught in function 'get_users', process 'MainProcess' (20440), thread 'MainThread' (20364):\nTraceback (most recent call last):\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\bot.py\", line 77, in <module>\n    bot.run(config.get_token)\n    │   │   │      └ <property object at 0x00000171E9D3DAD0>\n    │   │   └ <_config.Config object at 0x00000171E8387A30>\n    │   └ <function Client.run at 0x00000171E8E9D5A0>\n    └ <nextcord.ext.commands.bot.Bot object at 0x00000171E9B9FD60>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 796, in run\n    loop.run_forever()\n    │    └ <function ProactorEventLoop.run_forever at 0x00000171E8190D30>\n    └ <ProactorEventLoop running=True closed=False debug=False>\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\windows_events.py\", line 321, in run_forever\n    super().run_forever()\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\base_events.py\", line 600, in run_forever\n    self._run_once()\n    │    └ <function BaseEventLoop._run_once at 0x00000171E80DE0E0>\n    └ <ProactorEventLoop running=True closed=False debug=False>\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\base_events.py\", line 1896, in _run_once\n    handle._run()\n    │      └ <function Handle._run at 0x00000171E8071900>\n    └ <Handle <TaskStepMethWrapper object at 0x00000171EDA12AA0>()>\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n    │    │            │    │           │    └ <member '_args' of 'Handle' objects>\n    │    │            │    │           └ <Handle <TaskStepMethWrapper object at 0x00000171EDA12AA0>()>\n    │    │            │    └ <member '_callback' of 'Handle' objects>\n    │    │            └ <Handle <TaskStepMethWrapper object at 0x00000171EDA12AA0>()>\n    │    └ <member '_context' of 'Handle' objects>\n    └ <Handle <TaskStepMethWrapper object at 0x00000171EDA12AA0>()>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 445, in _run_event\n    await coro(*args, **kwargs)\n          │     │       └ {}\n          │     └ (<nextcord.interactions.Interaction object at 0x00000171E9D34EE0>,)\n          └ <bound method Client.on_interaction of <nextcord.ext.commands.bot.Bot object at 0x00000171E9B9FD60>>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 1872, in on_interaction\n    await self.process_application_commands(interaction)\n          │    │                            └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │    └ <function Client.process_application_commands at 0x00000171E8E9EE60>\n          └ <nextcord.ext.commands.bot.Bot object at 0x00000171E9B9FD60>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 1892, in process_application_commands\n    await app_cmd.call_from_interaction(interaction)\n          │       │                     └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │       └ <function BaseApplicationCommand.call_from_interaction at 0x00000171E8E0B2E0>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 2045, in call_from_interaction\n    await self.call(self._state, interaction)  # type: ignore\n          │    │    │    │       └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │    │    │    └ <nextcord.state.ConnectionState object at 0x00000171E9FCA290>\n          │    │    └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n          │    └ <function SlashApplicationCommand.call at 0x00000171E8E0BBE0>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 2397, in call\n    await self.call_slash(state, interaction, option_data)\n          │    │          │      │            └ [{'value': '877604100706025482', 'type': 8, 'name': 'guild_role'}]\n          │    │          │      └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │    │          └ <nextcord.state.ConnectionState object at 0x00000171E9FCA290>\n          │    └ <function SlashCommandMixin.call_slash at 0x00000171E8E0A830>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 1514, in call_slash\n    await self.invoke_callback_with_hooks(state, interaction, **kwargs)\n          │    │                          │      │              └ {'guild_role': <Role id=877604100706025482 name='Cryo'>}\n          │    │                          │      └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │    │                          └ <nextcord.state.ConnectionState object at 0x00000171E9FCA290>\n          │    └ <function CallbackMixin.invoke_callback_with_hooks at 0x00000171E8E09900>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 824, in invoke_callback_with_hooks\n    await self(interaction, *args, **kwargs)\n          │    │             │       └ {'guild_role': <Role id=877604100706025482 name='Cryo'>}\n          │    │             └ ()\n          │    └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n> File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\cogs\\master.py\", line 120, in get_users\n    f\"{self.get_timestamp(minutes=5, discord=True, style='T')}\"\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\slavebot\\utils.py\", line 126, in get_timestamp\n    days: timedelta = timedelta(days=\"\", hours=hours, minutes=minutes, seconds=seconds)\n    │                 │                        │              │                └ 0\n    │                 │                        │              └ 5\n    │                 │                        └ 0\n    │                 └ <class 'datetime.timedelta'>\n    └ ''\n\nTypeError: unsupported type for timedelta days component: str\n", "record": {"elapsed": {"repr": "0:00:08.096997", "seconds": 8.096997}, "exception": {"type": "TypeError", "value": "unsupported type for timedelta days component: str", "traceback": true}, "extra": {}, "file": {"name": "master.py", "path": "C:\\Users\\phili\\PycharmProjects\\GuildsBot\\cogs\\master.py"}, "function": "get_users", "level": {"icon": "❌", "name": "ERROR", "no": 40}, "line": 120, "message": "An error has been caught in function 'get_users', process 'MainProcess' (20440), thread 'MainThread' (20364):", "module": "master", "name": "cogs.master", "process": {"id": 20440, "name": "MainProcess"}, "thread": {"id": 20364, "name": "MainThread"}, "time": {"repr": "2022-07-14 08:04:10.484756+03:00", "timestamp": 1657775050.484756}}}
{"text": "2022-07-14T08:04:10.535757+0300 | ERROR      | An error has been caught in function 'get_users', process 'MainProcess' (20440), thread 'MainThread' (20364):\nTraceback (most recent call last):\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\bot.py\", line 77, in <module>\n    bot.run(config.get_token)\n    │   │   │      └ <property object at 0x00000171E9D3DAD0>\n    │   │   └ <_config.Config object at 0x00000171E8387A30>\n    │   └ <function Client.run at 0x00000171E8E9D5A0>\n    └ <nextcord.ext.commands.bot.Bot object at 0x00000171E9B9FD60>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 796, in run\n    loop.run_forever()\n    │    └ <function ProactorEventLoop.run_forever at 0x00000171E8190D30>\n    └ <ProactorEventLoop running=True closed=False debug=False>\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\windows_events.py\", line 321, in run_forever\n    super().run_forever()\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\base_events.py\", line 600, in run_forever\n    self._run_once()\n    │    └ <function BaseEventLoop._run_once at 0x00000171E80DE0E0>\n    └ <ProactorEventLoop running=True closed=False debug=False>\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\base_events.py\", line 1896, in _run_once\n    handle._run()\n    │      └ <function Handle._run at 0x00000171E8071900>\n    └ <Handle <TaskStepMethWrapper object at 0x00000171EDA12AA0>()>\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n    │    │            │    │           │    └ <member '_args' of 'Handle' objects>\n    │    │            │    │           └ <Handle <TaskStepMethWrapper object at 0x00000171EDA12AA0>()>\n    │    │            │    └ <member '_callback' of 'Handle' objects>\n    │    │            └ <Handle <TaskStepMethWrapper object at 0x00000171EDA12AA0>()>\n    │    └ <member '_context' of 'Handle' objects>\n    └ <Handle <TaskStepMethWrapper object at 0x00000171EDA12AA0>()>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 445, in _run_event\n    await coro(*args, **kwargs)\n          │     │       └ {}\n          │     └ (<nextcord.interactions.Interaction object at 0x00000171E9D34EE0>,)\n          └ <bound method Client.on_interaction of <nextcord.ext.commands.bot.Bot object at 0x00000171E9B9FD60>>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 1872, in on_interaction\n    await self.process_application_commands(interaction)\n          │    │                            └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │    └ <function Client.process_application_commands at 0x00000171E8E9EE60>\n          └ <nextcord.ext.commands.bot.Bot object at 0x00000171E9B9FD60>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 1892, in process_application_commands\n    await app_cmd.call_from_interaction(interaction)\n          │       │                     └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │       └ <function BaseApplicationCommand.call_from_interaction at 0x00000171E8E0B2E0>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 2045, in call_from_interaction\n    await self.call(self._state, interaction)  # type: ignore\n          │    │    │    │       └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │    │    │    └ <nextcord.state.ConnectionState object at 0x00000171E9FCA290>\n          │    │    └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n          │    └ <function SlashApplicationCommand.call at 0x00000171E8E0BBE0>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 2397, in call\n    await self.call_slash(state, interaction, option_data)\n          │    │          │      │            └ [{'value': '877604100706025482', 'type': 8, 'name': 'guild_role'}]\n          │    │          │      └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │    │          └ <nextcord.state.ConnectionState object at 0x00000171E9FCA290>\n          │    └ <function SlashCommandMixin.call_slash at 0x00000171E8E0A830>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 1514, in call_slash\n    await self.invoke_callback_with_hooks(state, interaction, **kwargs)\n          │    │                          │      │              └ {'guild_role': <Role id=877604100706025482 name='Cryo'>}\n          │    │                          │      └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │    │                          └ <nextcord.state.ConnectionState object at 0x00000171E9FCA290>\n          │    └ <function CallbackMixin.invoke_callback_with_hooks at 0x00000171E8E09900>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 824, in invoke_callback_with_hooks\n    await self(interaction, *args, **kwargs)\n          │    │             │       └ {'guild_role': <Role id=877604100706025482 name='Cryo'>}\n          │    │             └ ()\n          │    └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n> File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\cogs\\master.py\", line 120, in get_users\n    f\"{self.get_timestamp(minutes=5, discord=True, style='T')}\"\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\slavebot\\utils.py\", line 126, in get_timestamp\n    days: timedelta = timedelta(days=\"\", hours=hours, minutes=minutes, seconds=seconds)\n    │                 │                        │              │                └ 0\n    │                 │                        │              └ 5\n    │                 │                        └ 0\n    │                 └ <class 'datetime.timedelta'>\n    └ ''\n\nTypeError: unsupported type for timedelta days component: str\n", "record": {"elapsed": {"repr": "0:00:08.147998", "seconds": 8.147998}, "exception": {"type": "TypeError", "value": "unsupported type for timedelta days component: str", "traceback": true}, "extra": {}, "file": {"name": "master.py", "path": "C:\\Users\\phili\\PycharmProjects\\GuildsBot\\cogs\\master.py"}, "function": "get_users", "level": {"icon": "❌", "name": "ERROR", "no": 40}, "line": 120, "message": "An error has been caught in function 'get_users', process 'MainProcess' (20440), thread 'MainThread' (20364):", "module": "master", "name": "cogs.master", "process": {"id": 20440, "name": "MainProcess"}, "thread": {"id": 20364, "name": "MainThread"}, "time": {"repr": "2022-07-14 08:04:10.535757+03:00", "timestamp": 1657775050.535757}}}
{"text": "2022-07-14T08:04:10.612758+0300 | ERROR      | An error has been caught in function 'get_users', process 'MainProcess' (20440), thread 'MainThread' (20364):\nTraceback (most recent call last):\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\bot.py\", line 77, in <module>\n    bot.run(config.get_token)\n    │   │   │      └ <property object at 0x00000171E9D3DAD0>\n    │   │   └ <_config.Config object at 0x00000171E8387A30>\n    │   └ <function Client.run at 0x00000171E8E9D5A0>\n    └ <nextcord.ext.commands.bot.Bot object at 0x00000171E9B9FD60>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 796, in run\n    loop.run_forever()\n    │    └ <function ProactorEventLoop.run_forever at 0x00000171E8190D30>\n    └ <ProactorEventLoop running=True closed=False debug=False>\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\windows_events.py\", line 321, in run_forever\n    super().run_forever()\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\base_events.py\", line 600, in run_forever\n    self._run_once()\n    │    └ <function BaseEventLoop._run_once at 0x00000171E80DE0E0>\n    └ <ProactorEventLoop running=True closed=False debug=False>\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\base_events.py\", line 1896, in _run_once\n    handle._run()\n    │      └ <function Handle._run at 0x00000171E8071900>\n    └ <Handle <TaskStepMethWrapper object at 0x00000171EDA12AA0>()>\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n    │    │            │    │           │    └ <member '_args' of 'Handle' objects>\n    │    │            │    │           └ <Handle <TaskStepMethWrapper object at 0x00000171EDA12AA0>()>\n    │    │            │    └ <member '_callback' of 'Handle' objects>\n    │    │            └ <Handle <TaskStepMethWrapper object at 0x00000171EDA12AA0>()>\n    │    └ <member '_context' of 'Handle' objects>\n    └ <Handle <TaskStepMethWrapper object at 0x00000171EDA12AA0>()>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 445, in _run_event\n    await coro(*args, **kwargs)\n          │     │       └ {}\n          │     └ (<nextcord.interactions.Interaction object at 0x00000171E9D34EE0>,)\n          └ <bound method Client.on_interaction of <nextcord.ext.commands.bot.Bot object at 0x00000171E9B9FD60>>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 1872, in on_interaction\n    await self.process_application_commands(interaction)\n          │    │                            └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │    └ <function Client.process_application_commands at 0x00000171E8E9EE60>\n          └ <nextcord.ext.commands.bot.Bot object at 0x00000171E9B9FD60>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 1892, in process_application_commands\n    await app_cmd.call_from_interaction(interaction)\n          │       │                     └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │       └ <function BaseApplicationCommand.call_from_interaction at 0x00000171E8E0B2E0>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 2045, in call_from_interaction\n    await self.call(self._state, interaction)  # type: ignore\n          │    │    │    │       └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │    │    │    └ <nextcord.state.ConnectionState object at 0x00000171E9FCA290>\n          │    │    └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n          │    └ <function SlashApplicationCommand.call at 0x00000171E8E0BBE0>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 2397, in call\n    await self.call_slash(state, interaction, option_data)\n          │    │          │      │            └ [{'value': '877604100706025482', 'type': 8, 'name': 'guild_role'}]\n          │    │          │      └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │    │          └ <nextcord.state.ConnectionState object at 0x00000171E9FCA290>\n          │    └ <function SlashCommandMixin.call_slash at 0x00000171E8E0A830>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 1514, in call_slash\n    await self.invoke_callback_with_hooks(state, interaction, **kwargs)\n          │    │                          │      │              └ {'guild_role': <Role id=877604100706025482 name='Cryo'>}\n          │    │                          │      └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │    │                          └ <nextcord.state.ConnectionState object at 0x00000171E9FCA290>\n          │    └ <function CallbackMixin.invoke_callback_with_hooks at 0x00000171E8E09900>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 824, in invoke_callback_with_hooks\n    await self(interaction, *args, **kwargs)\n          │    │             │       └ {'guild_role': <Role id=877604100706025482 name='Cryo'>}\n          │    │             └ ()\n          │    └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n> File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\cogs\\master.py\", line 120, in get_users\n    f\"{self.get_timestamp(minutes=5, discord=True, style='T')}\"\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\slavebot\\utils.py\", line 126, in get_timestamp\n    days: timedelta = timedelta(days=\"\", hours=hours, minutes=minutes, seconds=seconds)\n    │                 │                        │              │                └ 0\n    │                 │                        │              └ 5\n    │                 │                        └ 0\n    │                 └ <class 'datetime.timedelta'>\n    └ ''\n\nTypeError: unsupported type for timedelta days component: str\n", "record": {"elapsed": {"repr": "0:00:08.224999", "seconds": 8.224999}, "exception": {"type": "TypeError", "value": "unsupported type for timedelta days component: str", "traceback": true}, "extra": {}, "file": {"name": "master.py", "path": "C:\\Users\\phili\\PycharmProjects\\GuildsBot\\cogs\\master.py"}, "function": "get_users", "level": {"icon": "❌", "name": "ERROR", "no": 40}, "line": 120, "message": "An error has been caught in function 'get_users', process 'MainProcess' (20440), thread 'MainThread' (20364):", "module": "master", "name": "cogs.master", "process": {"id": 20440, "name": "MainProcess"}, "thread": {"id": 20364, "name": "MainThread"}, "time": {"repr": "2022-07-14 08:04:10.612758+03:00", "timestamp": 1657775050.612758}}}
{"text": "2022-07-14T08:04:10.685761+0300 | ERROR      | An error has been caught in function 'get_users', process 'MainProcess' (20440), thread 'MainThread' (20364):\nTraceback (most recent call last):\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\bot.py\", line 77, in <module>\n    bot.run(config.get_token)\n    │   │   │      └ <property object at 0x00000171E9D3DAD0>\n    │   │   └ <_config.Config object at 0x00000171E8387A30>\n    │   └ <function Client.run at 0x00000171E8E9D5A0>\n    └ <nextcord.ext.commands.bot.Bot object at 0x00000171E9B9FD60>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 796, in run\n    loop.run_forever()\n    │    └ <function ProactorEventLoop.run_forever at 0x00000171E8190D30>\n    └ <ProactorEventLoop running=True closed=False debug=False>\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\windows_events.py\", line 321, in run_forever\n    super().run_forever()\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\base_events.py\", line 600, in run_forever\n    self._run_once()\n    │    └ <function BaseEventLoop._run_once at 0x00000171E80DE0E0>\n    └ <ProactorEventLoop running=True closed=False debug=False>\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\base_events.py\", line 1896, in _run_once\n    handle._run()\n    │      └ <function Handle._run at 0x00000171E8071900>\n    └ <Handle <TaskStepMethWrapper object at 0x00000171EDA12AA0>()>\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n    │    │            │    │           │    └ <member '_args' of 'Handle' objects>\n    │    │            │    │           └ <Handle <TaskStepMethWrapper object at 0x00000171EDA12AA0>()>\n    │    │            │    └ <member '_callback' of 'Handle' objects>\n    │    │            └ <Handle <TaskStepMethWrapper object at 0x00000171EDA12AA0>()>\n    │    └ <member '_context' of 'Handle' objects>\n    └ <Handle <TaskStepMethWrapper object at 0x00000171EDA12AA0>()>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 445, in _run_event\n    await coro(*args, **kwargs)\n          │     │       └ {}\n          │     └ (<nextcord.interactions.Interaction object at 0x00000171E9D34EE0>,)\n          └ <bound method Client.on_interaction of <nextcord.ext.commands.bot.Bot object at 0x00000171E9B9FD60>>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 1872, in on_interaction\n    await self.process_application_commands(interaction)\n          │    │                            └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │    └ <function Client.process_application_commands at 0x00000171E8E9EE60>\n          └ <nextcord.ext.commands.bot.Bot object at 0x00000171E9B9FD60>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 1892, in process_application_commands\n    await app_cmd.call_from_interaction(interaction)\n          │       │                     └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │       └ <function BaseApplicationCommand.call_from_interaction at 0x00000171E8E0B2E0>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 2045, in call_from_interaction\n    await self.call(self._state, interaction)  # type: ignore\n          │    │    │    │       └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │    │    │    └ <nextcord.state.ConnectionState object at 0x00000171E9FCA290>\n          │    │    └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n          │    └ <function SlashApplicationCommand.call at 0x00000171E8E0BBE0>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 2397, in call\n    await self.call_slash(state, interaction, option_data)\n          │    │          │      │            └ [{'value': '877604100706025482', 'type': 8, 'name': 'guild_role'}]\n          │    │          │      └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │    │          └ <nextcord.state.ConnectionState object at 0x00000171E9FCA290>\n          │    └ <function SlashCommandMixin.call_slash at 0x00000171E8E0A830>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 1514, in call_slash\n    await self.invoke_callback_with_hooks(state, interaction, **kwargs)\n          │    │                          │      │              └ {'guild_role': <Role id=877604100706025482 name='Cryo'>}\n          │    │                          │      └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │    │                          └ <nextcord.state.ConnectionState object at 0x00000171E9FCA290>\n          │    └ <function CallbackMixin.invoke_callback_with_hooks at 0x00000171E8E09900>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 824, in invoke_callback_with_hooks\n    await self(interaction, *args, **kwargs)\n          │    │             │       └ {'guild_role': <Role id=877604100706025482 name='Cryo'>}\n          │    │             └ ()\n          │    └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n> File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\cogs\\master.py\", line 120, in get_users\n    f\"{self.get_timestamp(minutes=5, discord=True, style='T')}\"\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\slavebot\\utils.py\", line 126, in get_timestamp\n    days: timedelta = timedelta(days=\"\", hours=hours, minutes=minutes, seconds=seconds)\n    │                 │                        │              │                └ 0\n    │                 │                        │              └ 5\n    │                 │                        └ 0\n    │                 └ <class 'datetime.timedelta'>\n    └ ''\n\nTypeError: unsupported type for timedelta days component: str\n", "record": {"elapsed": {"repr": "0:00:08.298002", "seconds": 8.298002}, "exception": {"type": "TypeError", "value": "unsupported type for timedelta days component: str", "traceback": true}, "extra": {}, "file": {"name": "master.py", "path": "C:\\Users\\phili\\PycharmProjects\\GuildsBot\\cogs\\master.py"}, "function": "get_users", "level": {"icon": "❌", "name": "ERROR", "no": 40}, "line": 120, "message": "An error has been caught in function 'get_users', process 'MainProcess' (20440), thread 'MainThread' (20364):", "module": "master", "name": "cogs.master", "process": {"id": 20440, "name": "MainProcess"}, "thread": {"id": 20364, "name": "MainThread"}, "time": {"repr": "2022-07-14 08:04:10.685761+03:00", "timestamp": 1657775050.685761}}}
{"text": "2022-07-14T08:04:10.747758+0300 | ERROR      | An error has been caught in function 'get_users', process 'MainProcess' (20440), thread 'MainThread' (20364):\nTraceback (most recent call last):\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\bot.py\", line 77, in <module>\n    bot.run(config.get_token)\n    │   │   │      └ <property object at 0x00000171E9D3DAD0>\n    │   │   └ <_config.Config object at 0x00000171E8387A30>\n    │   └ <function Client.run at 0x00000171E8E9D5A0>\n    └ <nextcord.ext.commands.bot.Bot object at 0x00000171E9B9FD60>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 796, in run\n    loop.run_forever()\n    │    └ <function ProactorEventLoop.run_forever at 0x00000171E8190D30>\n    └ <ProactorEventLoop running=True closed=False debug=False>\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\windows_events.py\", line 321, in run_forever\n    super().run_forever()\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\base_events.py\", line 600, in run_forever\n    self._run_once()\n    │    └ <function BaseEventLoop._run_once at 0x00000171E80DE0E0>\n    └ <ProactorEventLoop running=True closed=False debug=False>\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\base_events.py\", line 1896, in _run_once\n    handle._run()\n    │      └ <function Handle._run at 0x00000171E8071900>\n    └ <Handle <TaskStepMethWrapper object at 0x00000171EDA12AA0>()>\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n    │    │            │    │           │    └ <member '_args' of 'Handle' objects>\n    │    │            │    │           └ <Handle <TaskStepMethWrapper object at 0x00000171EDA12AA0>()>\n    │    │            │    └ <member '_callback' of 'Handle' objects>\n    │    │            └ <Handle <TaskStepMethWrapper object at 0x00000171EDA12AA0>()>\n    │    └ <member '_context' of 'Handle' objects>\n    └ <Handle <TaskStepMethWrapper object at 0x00000171EDA12AA0>()>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 445, in _run_event\n    await coro(*args, **kwargs)\n          │     │       └ {}\n          │     └ (<nextcord.interactions.Interaction object at 0x00000171E9D34EE0>,)\n          └ <bound method Client.on_interaction of <nextcord.ext.commands.bot.Bot object at 0x00000171E9B9FD60>>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 1872, in on_interaction\n    await self.process_application_commands(interaction)\n          │    │                            └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │    └ <function Client.process_application_commands at 0x00000171E8E9EE60>\n          └ <nextcord.ext.commands.bot.Bot object at 0x00000171E9B9FD60>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 1892, in process_application_commands\n    await app_cmd.call_from_interaction(interaction)\n          │       │                     └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │       └ <function BaseApplicationCommand.call_from_interaction at 0x00000171E8E0B2E0>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 2045, in call_from_interaction\n    await self.call(self._state, interaction)  # type: ignore\n          │    │    │    │       └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │    │    │    └ <nextcord.state.ConnectionState object at 0x00000171E9FCA290>\n          │    │    └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n          │    └ <function SlashApplicationCommand.call at 0x00000171E8E0BBE0>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 2397, in call\n    await self.call_slash(state, interaction, option_data)\n          │    │          │      │            └ [{'value': '877604100706025482', 'type': 8, 'name': 'guild_role'}]\n          │    │          │      └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │    │          └ <nextcord.state.ConnectionState object at 0x00000171E9FCA290>\n          │    └ <function SlashCommandMixin.call_slash at 0x00000171E8E0A830>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 1514, in call_slash\n    await self.invoke_callback_with_hooks(state, interaction, **kwargs)\n          │    │                          │      │              └ {'guild_role': <Role id=877604100706025482 name='Cryo'>}\n          │    │                          │      └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │    │                          └ <nextcord.state.ConnectionState object at 0x00000171E9FCA290>\n          │    └ <function CallbackMixin.invoke_callback_with_hooks at 0x00000171E8E09900>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 824, in invoke_callback_with_hooks\n    await self(interaction, *args, **kwargs)\n          │    │             │       └ {'guild_role': <Role id=877604100706025482 name='Cryo'>}\n          │    │             └ ()\n          │    └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n> File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\cogs\\master.py\", line 120, in get_users\n    f\"{self.get_timestamp(minutes=5, discord=True, style='T')}\"\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\slavebot\\utils.py\", line 126, in get_timestamp\n    days: timedelta = timedelta(days=\"\", hours=hours, minutes=minutes, seconds=seconds)\n    │                 │                        │              │                └ 0\n    │                 │                        │              └ 5\n    │                 │                        └ 0\n    │                 └ <class 'datetime.timedelta'>\n    └ ''\n\nTypeError: unsupported type for timedelta days component: str\n", "record": {"elapsed": {"repr": "0:00:08.359999", "seconds": 8.359999}, "exception": {"type": "TypeError", "value": "unsupported type for timedelta days component: str", "traceback": true}, "extra": {}, "file": {"name": "master.py", "path": "C:\\Users\\phili\\PycharmProjects\\GuildsBot\\cogs\\master.py"}, "function": "get_users", "level": {"icon": "❌", "name": "ERROR", "no": 40}, "line": 120, "message": "An error has been caught in function 'get_users', process 'MainProcess' (20440), thread 'MainThread' (20364):", "module": "master", "name": "cogs.master", "process": {"id": 20440, "name": "MainProcess"}, "thread": {"id": 20364, "name": "MainThread"}, "time": {"repr": "2022-07-14 08:04:10.747758+03:00", "timestamp": 1657775050.747758}}}
{"text": "2022-07-14T08:04:10.812761+0300 | ERROR      | An error has been caught in function 'get_users', process 'MainProcess' (20440), thread 'MainThread' (20364):\nTraceback (most recent call last):\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\bot.py\", line 77, in <module>\n    bot.run(config.get_token)\n    │   │   │      └ <property object at 0x00000171E9D3DAD0>\n    │   │   └ <_config.Config object at 0x00000171E8387A30>\n    │   └ <function Client.run at 0x00000171E8E9D5A0>\n    └ <nextcord.ext.commands.bot.Bot object at 0x00000171E9B9FD60>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 796, in run\n    loop.run_forever()\n    │    └ <function ProactorEventLoop.run_forever at 0x00000171E8190D30>\n    └ <ProactorEventLoop running=True closed=False debug=False>\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\windows_events.py\", line 321, in run_forever\n    super().run_forever()\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\base_events.py\", line 600, in run_forever\n    self._run_once()\n    │    └ <function BaseEventLoop._run_once at 0x00000171E80DE0E0>\n    └ <ProactorEventLoop running=True closed=False debug=False>\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\base_events.py\", line 1896, in _run_once\n    handle._run()\n    │      └ <function Handle._run at 0x00000171E8071900>\n    └ <Handle <TaskStepMethWrapper object at 0x00000171EDA12AA0>()>\n\n  File \"C:\\Users\\phili\\AppData\\Local\\Programs\\Python\\Python310\\lib\\asyncio\\events.py\", line 80, in _run\n    self._context.run(self._callback, *self._args)\n    │    │            │    │           │    └ <member '_args' of 'Handle' objects>\n    │    │            │    │           └ <Handle <TaskStepMethWrapper object at 0x00000171EDA12AA0>()>\n    │    │            │    └ <member '_callback' of 'Handle' objects>\n    │    │            └ <Handle <TaskStepMethWrapper object at 0x00000171EDA12AA0>()>\n    │    └ <member '_context' of 'Handle' objects>\n    └ <Handle <TaskStepMethWrapper object at 0x00000171EDA12AA0>()>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 445, in _run_event\n    await coro(*args, **kwargs)\n          │     │       └ {}\n          │     └ (<nextcord.interactions.Interaction object at 0x00000171E9D34EE0>,)\n          └ <bound method Client.on_interaction of <nextcord.ext.commands.bot.Bot object at 0x00000171E9B9FD60>>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 1872, in on_interaction\n    await self.process_application_commands(interaction)\n          │    │                            └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │    └ <function Client.process_application_commands at 0x00000171E8E9EE60>\n          └ <nextcord.ext.commands.bot.Bot object at 0x00000171E9B9FD60>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\client.py\", line 1892, in process_application_commands\n    await app_cmd.call_from_interaction(interaction)\n          │       │                     └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │       └ <function BaseApplicationCommand.call_from_interaction at 0x00000171E8E0B2E0>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 2045, in call_from_interaction\n    await self.call(self._state, interaction)  # type: ignore\n          │    │    │    │       └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │    │    │    └ <nextcord.state.ConnectionState object at 0x00000171E9FCA290>\n          │    │    └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n          │    └ <function SlashApplicationCommand.call at 0x00000171E8E0BBE0>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 2397, in call\n    await self.call_slash(state, interaction, option_data)\n          │    │          │      │            └ [{'value': '877604100706025482', 'type': 8, 'name': 'guild_role'}]\n          │    │          │      └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │    │          └ <nextcord.state.ConnectionState object at 0x00000171E9FCA290>\n          │    └ <function SlashCommandMixin.call_slash at 0x00000171E8E0A830>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 1514, in call_slash\n    await self.invoke_callback_with_hooks(state, interaction, **kwargs)\n          │    │                          │      │              └ {'guild_role': <Role id=877604100706025482 name='Cryo'>}\n          │    │                          │      └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          │    │                          └ <nextcord.state.ConnectionState object at 0x00000171E9FCA290>\n          │    └ <function CallbackMixin.invoke_callback_with_hooks at 0x00000171E8E09900>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\venv\\lib\\site-packages\\nextcord\\application_command.py\", line 824, in invoke_callback_with_hooks\n    await self(interaction, *args, **kwargs)\n          │    │             │       └ {'guild_role': <Role id=877604100706025482 name='Cryo'>}\n          │    │             └ ()\n          │    └ <nextcord.interactions.Interaction object at 0x00000171E9D34EE0>\n          └ <nextcord.application_command.SlashApplicationCommand object at 0x00000171EA000460>\n\n> File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\cogs\\master.py\", line 120, in get_users\n    f\"{self.get_timestamp(minutes=5, discord=True, style='T')}\"\n\n  File \"C:\\Users\\phili\\PycharmProjects\\GuildsBot\\slavebot\\utils.py\", line 126, in get_timestamp\n    days: timedelta = timedelta(days=\"\", hours=hours, minutes=minutes, seconds=seconds)\n    │                 │                        │              │                └ 0\n    │                 │                        │              └ 5\n    │                 │                        └ 0\n    │                 └ <class 'datetime.timedelta'>\n    └ ''\n\nTypeError: unsupported type for timedelta days component: str\n", "record": {"elapsed": {"repr": "0:00:08.425002", "seconds": 8.425002}, "exception": {"type": "TypeError", "value": "unsupported type for timedelta days component: str", "traceback": true}, "extra": {}, "file": {"name": "master.py", "path": "C:\\Users\\phili\\PycharmProjects\\GuildsBot\\cogs\\master.py"}, "function": "get_users", "level": {"icon": "❌", "name": "ERROR", "no": 40}, "line": 120, "message": "An error has been caught in function 'get_users', process 'MainProcess' (20440), thread 'MainThread' (20364):", "module": "master", "name": "cogs.master", "process": {"id": 20440, "name": "MainProcess"}, "thread": {"id": 20364, "name": "MainThread"}, "time": {"repr": "2022-07-14 08:04:10.812761+03:00", "timestamp": 1657775050.812761}}}
